tests/console/mutt.pm

```diff
@@ -53,7 +53,7 @@ sub run {
     assert_script_run 'sed -i -e "/ssl_ca_certificates_file/d" ~/.muttrc' if is_sle('<=12-SP2');
 
     record_info 'receive mail', 'Run mutt as a user to read the mail';
-    enter_cmd "mutta";
+    enter_cmd "mutt";
```

Comment:
it was not a typo but previously it was `type_string "mutt\na"` so likely we need another `send_key` after that line

Reply:
let's see if we need this fix, see #12297

Thanks for trying to fix the problem but I guess a verification and inviting me to review would be nice.